internal/xds/translator/translator.go

```diff
@@ -768,10 +768,29 @@ func buildXdsUpstreamTLSCASecret(tlsConfig *ir.TLSUpstreamConfig) *tlsv3.Secret
 func buildXdsUpstreamTLSSocketWthCert(tlsConfig *ir.TLSUpstreamConfig) (*corev3.TransportSocket, error) {
 
 	var tlsCtx *tlsv3.UpstreamTlsContext
+	var tlsParams *tlsv3.TlsParameters
```

Comment:
can we reuse https://github.com/envoyproxy/gateway/blob/27a770f4db8ff275591934f4bc3080ba0d1aaddc/internal/xds/translator/listener.go#L572 ?

Reply:
/retest